scripts/variantstore/wdl/ImportGenomes.wdl

```diff
@@ -338,6 +350,9 @@ task LoadTable {
         echo "no ${FILES} files to process in $DIR"
     fi
 
+    # remove load lock
+    echo "Removing load lock"
+    gsutil rm "${DIR}${LOCKFILE}" || exit 1
```

Comment:
just wondering how we deal with this file still being there if the workflow crashes?  Even if it's just a "here's why I'm not worried"

Reply:
I think it's a manual process - you have to go look and figure out what you want to do. if you do want to re-run, you'd manually delete the lockfile. but definitely open to other ideas for processes? we talked a little bit in standup about how we could try to get fancy and infer the state of things, but landed that for now it could be a manual process of just preventing the bad thing (double loading) and requiring some manual investigation & remedying.

Reply:
maybe just paraphrase some of that and leave it as a comment for our future selves